docs/content/modeling/contextual-time-based-authorization.mdx

```diff
@@ -36,6 +36,13 @@ Contextual Tuples should be used when modeling cases where a user's access to an
 
 To follow this guide, you should be familiar with some <ProductConcept />.
 
+### <ProductName format={ProductNameFormat.ShortForm}/> Concepts
+
+- A <ProductConcept section="what-is-a-relation" linkName="Relation" />: is a string defined in the type definition of an authorization model that defines the possibility of a relationship between objects of this type and other types.
```

Comment:
> objects of this type and other types.

What is this here?

Reply:
This is from https://openfga.dev/docs/concepts#what-is-a-relation.  In the context of relation, we are discussing relationship between two types.

Reply:
I mean there is no subject for "this" in that sentence. I think it should be something like:

> A Relation is a string defined in the type definition of an authorization model that defines the possibility of a relationship between an object and a user.

Or something like that.

Reply:
Interesting... in this case, we should update all relation definition to reflect the new definition.

Reply:
If I were to change it it would be "between objects of that type" referencing the type definition we are talking about. It is not talking about possible relations across all objects of all types in the system

Reply:
See the change made